src/main/java/com/mindee/model/documenttype/DocumentLevelResponse.java

```diff
@@ -0,0 +1,5 @@
+package com.mindee.model.documenttype;
+
+public interface DocumentLevelResponse {
```

Comment:
What is the purpose of this empty interface ?

Reply:
it is a marker and gives us the ability to add future default/implemented methods as well. Right now it helps us ensure that the right type of classes are used with deserializer - 

public class DocumentResponseDeserializer<T extends BaseDocumentResponse & PredictionApiResponse<S,U>,
  S extends DocumentLevelResponse, U extends PageLevelResponse> extends StdDeserializer<T>